### ISSUE TITLE
Add more analysis Graphs and improve over the current ones

### ISSUE THREAD

**Issue by shrinish123** [shrinish123/lets-Upsolve#3]
Right now the plan is to add graph plots for no questions solved rating-wise, bar graphs for index wise questions, languages pie chart, and also breed some insightful numbers for performance improvement on code forces

**Comment by shrinish123** [comment]
Hello @shaunak-09, I will assign this issue to you. But please give me a date by which you would make your 1st PR. You don't have to complete everything, you can make PRs in small steps by mentioning them. 
Talking about the graphs, you can always check out this site [here](https://cfviz.netlify.app/). So we don't plan to have everything on the website, like compare, etc. We aim for our app for our own improvement alone, So we plan to implement a similar type of homepage of that website alone on our analysis page. 
Right now you can go with any of the graphs you would want to but would recommend making the tables named 'Some numbers about ' and 'Contests of 'from that website. I don't recommend exactly the same but somewhat similar with some extra improvements maybe from your side. Although not necessary to do all this in the 1st step itself, you can start by just replicating it too, and then further on we can improve it. 
Also, a great help to me, if you are able to find which charts library that is being used on that website, so we can use it over here. Also if you have any good suggestions for the charts libraries are welcome.

**Comment by shaunak-09** [comment]
Thanks. I'll implement these two graphs by 17 Dec. I think chart js will be a good library to use in this case. Can u pls send me an screenshot of the 'contests of' graph as I am not able to see it.

**Comment by shrinish123** [comment]
Hello I think a good chart library I found is [Nivo Graphs](https://nivo.rocks/). I think it would be also easy to implement and rpahs and customizations are pretty good. 
also here's a screenshot you requested of : 

![graphs](https://user-images.githubusercontent.com/56491104/207256904-5cb3b5bd-f299-4727-948a-194962c9736c.png)

**Comment by shrinish123** [comment]
Hello any progress there @shaunak-09 ?

**Comment by shaunak-09** [comment]
Yeah I have implemented 2 bar graphs for now but I am facing some issues connecting the fetched data with the graph but by tomorrow I'll get it done.
